src/main/java/io/stargate/sgv2/jsonapi/exception/playing/DatabaseException.java

```diff
@@ -1,18 +1,21 @@
 package io.stargate.sgv2.jsonapi.exception.playing;
 
 public class DatabaseException extends ServerException {
+
+  public static final Scope SCOPE = Scope.DATABASE;
```

Comment:
agree it can be better, keeping visible and public as something to improve and have this issue tracking it https://github.com/stargate/data-api/issues/1377